queries/selectBins1To5OfZoomlevel1.rq

```diff
@@ -0,0 +1,24 @@
+PREFIX vg: <http://biohackathon.org/resource/vg#>
+PREFIX faldo: <http://biohackathon.org/resource/faldo#>
+PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
+
+SELECT ?cell ?cellregion ?inversionpercent ?positionpercent ?beginpos ?endpos ?path
+WHERE {
+        ?zoomlevel a vg:ZoomLevel;
+                vg:components ?components;
+                vg:zoomFactor ?zoomfactor .
+        FILTER(?zoomfactor = 1)
+        ?components vg:bins ?bin .
+        ?bin vg:cells ?cell;
+                vg:binRank ?binrank .
+        FILTER(?binrank < 6 && ?binrank > 0)
+        ?cell vg:cellRegions ?faldoregion;
+                vg:inversionPercent ?inversionpercent;
+                vg:positionPercent ?positionpercent .
+        ?faldoregion faldo:begin ?begin;
+                faldo:end ?end .
+        ?begin faldo:position ?beginpos .
+        ?end faldo:position ?endpos .
+        ?begin faldo:reference ?reference .
+        # THIS DOES NOT WORK BUT I DON'T KNOW WHY ?reference rdfs:label ?path .
```

Comment:
That's strange.. How about `?reference vg:path ?path .`?

Reply:
No. The `?reference` itself aready is a `vg:path`, so I get no output.

Reply:
We might miss `rdfs:label` for `vg:path` in the input turtle file?

Reply:
After some thinking I came to the following conclusion: Our ontology would allow us to do `?reference rdfs:label ?path .`, but we currently do not emit the `rdfs:lable` of any class into the Turtle. We might want to add that at some point.

Reply:
SpOdgi misses that, too.

Reply:
Thanks. If we don't have to query `vg:path` right now, we can leave as it is.